static/app/views/performance/newTraceDetails/traceDrawer/details/span/sections/description.tsx

```diff
@@ -76,6 +77,12 @@ export function SpanDescription({
             spanSlug: {op: span.op, group: groupHash},
             projectID: event.projectID,
           })}
+          onClick={() =>
+            trackAnalytics('trace.trace_layout.view_span_summary', {
+              organization,
+              module: resolvedModule,
+            })
+          }
```

Comment:
depending on whether `hasNewSpansUIFlag` resolves to true or not, you'll need to conditionally fire off the event for `view_span_summary` or `view_similar_spans`